scss/titles.scss

```diff
@@ -0,0 +1,20 @@
+.tech-title {
+  align-items: center;
+  display: flex;
+  font-size: $text-big;
+  height: 90px;
+  padding-left: 52px;
```

Comment:
Instead of padding the main element is better to move the decorative one.

Reply:
Padding in the main element is removed and translateX is added in background image.